From 598813a7316ec153c109f112b47b8812c1205e91 Mon Sep 17 00:00:00 2001
From: Ley Foon Tan <lftan@altera.com>
Date: Mon, 5 Aug 2013 16:33:24 +0800
Subject: [PATCH 042/142] FogBugz #143451: Fix mutex compatible string

Fix mutex compatible string from mutex-1.0 to hwmutex-1.0.

Signed-off-by: Ley Foon Tan <lftan@altera.com>
[Original patch taken from
 https://github.com/altera-opensource/linux-socfpga.git
 tag rel_socfpga-4.8_17.03.01_pr]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/misc/altera_hwmutex.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/altera_hwmutex.c b/drivers/misc/altera_hwmutex.c
index 2bfdc61e36d5..43e4e564e9f1 100644
--- a/drivers/misc/altera_hwmutex.c
+++ b/drivers/misc/altera_hwmutex.c
@@ -288,7 +288,7 @@ static int altera_mutex_remove(struct platform_device *pdev)
 }
 
 static const struct of_device_id altera_mutex_match[] = {
-	{ .compatible = "altr,mutex-1.0" },
+	{ .compatible = "altr,hwmutex-1.0" },
 	{ /* Sentinel */ }
 };
 
-- 
2.11.0

